### DISCUSSION TITLE
`code-link` in other (non-html) formats?

### DISCUSSION THREAD

**Discussion by eitsupi** [quarto-dev/quarto-cli#3777] [Feature Requests]
The automatic linking by the `downlit` R package is really cool.
At the moment it seems to be only available in HTML format in Quarto, but I am wondering if it is possible to use this in Markdown format.
(I may be wrong as I have not used it, but it seems to be used in `hugodown`?)
https://quarto.org/docs/output-formats/html-code.html#code-linking

My use case is to generate md files from qmd to pass to other site generators, and it would be great if I could automatically link R functions to reference pages.

**Comment by mcanouil** [comment]
`downlit` can indeed do what you want with <https://downlit.r-lib.org/reference/downlit_md_path.html>.
Thus, Quarto could make the `code-link` option work for markdown formats as well.
(I had to double check the documentation because I really thought this was already the case)

**Reply by eitsupi** [reply]
Yeah, I was actually surprised to find this doesn't work when I used it myself.
